json_serializable/pubspec.yaml

```diff
@@ -8,6 +8,7 @@ environment:
 dependencies:
```

Comment:
@natebosch – should update the SDK dependency then, too? Right?

Reply:
IMO it shouldn't be a requirement. It's pub's job to take dependency SDK constraints into account when version solving.

If I'm only using stuff from `package:foo` version `1.0.0` but my dependency tightened the constraint to `1.1.0` I don't go hunting for that information and bump tighten my constraint too - that's what the version solve is for. From my perspective the SDK constraint is just like a package constraint in this way.

In practice I think it's fine to do it since we're not testing on older SDKs so we're not confident that we haven't suddenly stopped working for reasons around _this_ package.

Reply:
Done